opensearch/src/test/java/org/opensearch/sql/opensearch/storage/script/filter/lucene/MatchQueryTest.java

```diff
@@ -112,21 +116,27 @@ static Stream<List<Expression>> generateValidData() {
   @ParameterizedTest
   @MethodSource("generateValidData")
   public void test_valid_parameters(List<Expression> validArgs) {
-    Assertions.assertNotNull(matchQuery.build(new MatchExpression(validArgs)));
+    for (FunctionName funcName : functionNames) {
```

Comment:
It would be better not to bundle tests together - especially for the sunny-day scenario.  
If one of these fails, it's harder to understand what caused the error... it will require debugging.  Rather than if you split these into separate tests, you would know right away.

Reply:
Fixed in 990e2f32d68f948ec7efc012813978f107bca2b0